chiselFrontend/src/main/scala/Chisel/When.scala

```diff
@@ -2,9 +2,12 @@
 
 package Chisel
```

Comment:
All of these return Unit or things that cannot be applied. Why do they need macro transforms?